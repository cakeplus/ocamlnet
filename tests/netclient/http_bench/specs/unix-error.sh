. helpers.sh

server_port=`./refuser`
request -get / -catch -run
