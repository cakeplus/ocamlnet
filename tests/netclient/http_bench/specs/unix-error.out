Http_protocol: Unix.Unix_error(ECONNREFUSED, "connect_check", "")
